namespaces/cloud-platform-live-0.k8s.integration.dsd.io/reference-app-development/00-namespace.yaml

```diff
@@ -1,12 +1,12 @@
 apiVersion: v1
 kind: Namespace
 metadata:
-  name: reference-app
+  name: reference-app-development
   labels:
     cloud-platform.justice.gov.uk/is-production: "false"
     cloud-platform.justice.gov.uk/environment-name: "development"
   annotations:
-    cloud-platform.justice.gov.uk/business-unit: "cloud-platform"
-    cloud-platform.justice.gov.uk/application: "Reference App"
-    cloud-platform.justice.gov.uk/owner: "Cloud Platform: platforms@digital.justice.gov.uk"
+    cloud-platform.justice.gov.uk/business-unit: "digital"
```

Comment:
So it turns out we have both done this wrong:

<img width="794" alt="screen shot 2018-10-17 at 13 16 38" src="https://user-images.githubusercontent.com/3952074/47085575-eb11fd80-d20e-11e8-9272-a8ebc9302a4e.png">

I think that means the right answer is:

```suggestion
    cloud-platform.justice.gov.uk/business-unit: "HQ"
```